docs/source/getting_started.rst

```diff
@@ -151,9 +133,9 @@ We also use the SciKit-learn wrapper for linear regression. The implementation i
 
     regressor_power_statistics = SKLearnWrapper(module=LinearRegression(fit_intercept=True))(shift1=shift_power_statistics,
                                                                                              shift2=shift_power_statistics2,
-                                                                                             month=calendar_month,
-                                                                                             weeday=calendar_weekday,
-                                                                                             weekend=calendar_weekend],
+                                                                                             month=CalendarFeature.month,
+                                                                                             weekday=CalendarFeature.weekday,
+                                                                                             weekend=CalendarFeature.weekend],
```

Comment:
It should be:
```
regressor_power_statistics = SKLearnWrapper(module=LinearRegression(fit_intercept=True))(shift1=shift_power_statistics,
                                                                                         shift2=shift_power_statistics2,
                                                                                         calendar=calendar,
                                                                                         target_power=scale_power_statistics
```

Since the output of calendar contains already calendar_month, calendar_weekday, and calendar_weekend. 

Can you also change this in the following paragraph?

Reply:
Fixed it and also changed the paragraph after it (and had to correct a small typo I made).